src/LanguageServer/Test/DiagnosticsTests.cs

```diff
@@ -116,131 +117,154 @@ public void TestInitialize()
             var doc = analysis.Document;
 
             var clientApp = Services.GetService<IClientApplication>();
-            var idle = Services.GetService<IIdleTimeService>();
-
-            var expected = 1;
+            var reported = new List<PublishDiagnosticsParams>();
```

Comment:
This test also seems to be flaky...

```
Test Name:	Publish
Test FullName:	Microsoft.Python.LanguageServer.Tests.DiagnosticsTests.Publish
Test Source:	C:\Users\jabaile\mikhail-language-server\src\LanguageServer\Test\DiagnosticsTests.cs : line 113
Test Outcome:	Failed
Test Duration:	0:00:03.6129852

Result StackTrace:	
at FluentAssertions.Execution.LateBoundTestFramework.Throw(String message) in C:\projects\fluentassertions-vf06b\Src\FluentAssertions\Execution\LateBoundTestFramework.cs:line 16
   at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args) in C:\projects\fluentassertions-vf06b\Src\FluentAssertions\Execution\AssertionScope.cs:line 225
   at FluentAssertions.Numeric.NumericAssertions`1.Be(T expected, String because, Object[] becauseArgs) in C:\projects\fluentassertions-vf06b\Src\FluentAssertions\Numeric\NumericAssertions.cs:line 50
   at Microsoft.Python.LanguageServer.Tests.DiagnosticsTests.Publish() in C:\Users\jabaile\mikhail-language-server\src\LanguageServer\Test\DiagnosticsTests.cs:line 138
Result Message:	Expected reported.Count to be 1, but found 0.
```

Reply:
Same as above, needs `await doc.GetAstAsync();`